contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -388,18 +374,19 @@ abstract contract FunctionsBilling is HasRouter, IFunctionsBilling {
     // (1e18 juels/link) * (wei/gas) / (wei/link) = juels per gas
     uint256 juelsPerGas = (1e18 * tx.gasprice) / uint256(weiPerUnitLink);
     // Gas overhead without callback
-    uint96 gasOverheadJuels = uint96(juelsPerGas * commitment.gasOverhead);
-    uint96 costWithoutFulfillment = gasOverheadJuels + commitment.donFee;
+    uint96 gasOverheadJuels = uint96(
+      juelsPerGas * (commitment.gasOverheadBeforeCallback + commitment.gasOverheadAfterCallback)
+    );
 
     // The Functions Router will perform the callback to the client contract
     IFunctionsRouter router = IFunctionsRouter(address(_getRouter()));
     (uint8 result, uint96 callbackCostJuels) = router.fulfill(
-      requestId,
       response,
       err,
       uint96(juelsPerGas),
-      costWithoutFulfillment,
-      msg.sender
+      gasOverheadJuels + commitment.donFee, // costWithoutFulfillment
```

Comment:
Can costWithoutFulfillment also include the admin fee?  Then we don't have to worry about accounting for the admin fee in the router's `fulfill` method.  I think this simplifies the code, but correct me if I am mistaken.

Reply:
Then we would be trusting the coordinator to tell the Router what to charge as the admin Fee, rather than taking it from the commitment data